Content.Shared/Chat/ChatModifiers/InsertLoCChatModifier.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Linq;
+using Content.Shared.CCVar;
+using Content.Shared.Decals;
+using Robust.Shared.Configuration;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+
+namespace Content.Shared.Chat.ChatModifiers;
+
+/// <summary>
+/// Inserts an LoC string after a specific node. Useful for formatting certain messages, such as whispering.
+/// </summary>
+[Serializable]
+[DataDefinition]
+public sealed partial class InsertLoCChatModifier : ChatModifier
+{
+    /// <summary>
+    /// If false, the string will be inserted before the node.
+    /// </summary>
+    [DataField]
+    public bool AfterNode = true;
```

Comment:
a little bit ugly and counterintuitive, maybe base class fro insert and insertAfter/insertBefore derived types would be better? so you would just hide this stuff from ppl who write yaml.